src/main/resources/e-canteen.jdl

```diff
@@ -0,0 +1,247 @@
+
+entity Worker { // any institution we'll be dealing with
```

Comment:
It is better to merge Worker/Student/Parent as one entity which is called User and separate them by Role

Reply:
question: User could have relation (many to many) with itself?